game.js

```diff
@@ -280,7 +280,7 @@ class Game {
     this.tickSize = 40;
     this.generation = 0;
     this.eventListeners = new EventListeners();
-
+    this.isRunning = false;
```

Comment:
why did you add this here? The reason why I added this to the reset function is so that you don't have to initialize the value twice.